src/emulator/databaseEmulator.ts

```diff
@@ -153,7 +153,13 @@ export class DatabaseEmulator implements EmulatorInstance {
     });
   }
 
-  private async updateRules(instance: string, content: string): Promise<any> {
+  private async updateRules(instance: string, rulesPath: string): Promise<any> {
+    const rulesExt = path.extname(rulesPath);
+    const content =
+      rulesExt === ".bolt"
+        ? parseBoltRules(rulesPath).toString()
+        : fs.readFileSync(rulesPath, "utf8").toString();
```

Comment:
IIRC, `readFileSync(path, "utf8")` returns a string, so `.toString()`-ing it is redundant

```suggestion
        : fs.readFileSync(rulesPath, "utf8");
```